### PR TITLE
WTW-531, WTW-532, WTW-533 Implements new icons

### PR DESCRIPTION
**Pull request by bombozama**
Full list of available icons:
- AccessibleBathroom
- AccessibleEquipment
- AccessiblePaths
- AccessibleTransportation
- Accommodation
- Activities
- AdaptiveEquipment
- AirConditioner
- AirportShuttle
- AlcoholicBeverages
- Assistance
- Atm
- AudioGuides
- Auditorium
- AuditoryGuidance
- Baby
- Backpacker
- Baggage
- Bar
- Bathroom
- BathroomAccessInterior
- BathroomInterior
- BathroomSink
- BeachAccess
- BeachFront
- Bed
- BedBreakfast
- BedroomAccess
- BeerBar
- Blind
- Boat
- Braille
- Breakfast
- BuildingEntrance
- Bus
- BusinessCenter
- Cabin
- Cafeteria
- Calendar
- Car
- Chair
- Champagne
- ChangingRoom
- Clock
- Closet
- CoffeeMaker
- CommonBathroom
- Complete
- Conference
- Crutches
- CulturalPlace
- Culture
- CurrencyExchange
- Daycare
- Deaf
- Dinner
- Done
- DryCleaning
- ElectricDeviceWheelchair
- Elevator
- Entertainment
- Entrance
- Equipment
- EventRoom
- FitnessCenter
- FoldablePowerWheelchair
- FoodService
- GeneralAttribute
- Gratuities
- GroundTransportation
- Guides
- Gym
- Health
- HearingAid
- HeatedPool
- HeavyPowerWheelchair
- Helmet
- HighBathtub
- Hotel
- Housekeeping
- Id
- Instructor
- Insurance
- Interior
- JeepCar
- Key
- Keys
- LakeSea
- LandActivity
- Language
- Laundry
- Level
- Lifevest
- Lobby
- Location
- LowerBathtub
- LowerSteps
- Lunch
- MeasuringTape
- MeetingPoint
- MiniFridge
- Minus
- Missing
- MobilePhone
- Money
- Motel
- MultiDayTrip
- Next
- No
- Notification
- NotificationActive
- Nurse
- OneStep
- Other
- Outdoor
- Park
- Parking
- Pending
- Person
- PetFriendly
- PicnicTable
- Places
- PlacesToStay
- Plane
- Playground
- Plus
- PrivateTour
- Profile
- QueenBed
- Reception
- Resort
- Restaurant
- Restrictions
- RollInShower
- RoomEntrance
- RoomService
- Safe
- SafetyEquipment
- Sauna
- Save
- Scooter
- SedanCar
- Share
- Shop
- Shopping
- Shower
- ShowerGrabBar
- ShowerLowStep
- ShowerSeat
- Sink
- SinkTap
- SmokeFree
- SnackBar
- Snacks
- SocialDistancing
- SoftDrink
- Spa
- Stairs
- StepFree
- SwimmingPool
- Switch
- TV
- Taxes
- Tea
- TeaRoom
- Terrace
- ThingsToDo
- ThreeSteps
- Ticket
- Toilet
- ToiletGrabBar
- ToiletPaper
- TourGuide
- ToursActivities
- Transport
- Transportation
- TurningRadius
- TwentyFourHour
- ValetParking
- VendingMachine
- Verified
- WTWIso
- WTWLogo
- WTWLogoVertical
- Walker
- WalkingAid
- WaterActivity
- WaterBottle
- WaterFountain
- Wheelchair
- WheelchairRamp
- Wifi
- Work

### CODE REVIEW

**Review by martinmunillas — APPROVED** [review]
Did you check in the storybooks or somewhere that the viewport is appropiate for all the icons?

File: src/react/icons/AccessibleBathroom.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import Icon from '../Icon';
+import { IconComponent } from '../types';
+
+const AccessibleBathroom: IconComponent = (props: any) => (
+    <Icon viewBox="0 0 40 40" {...props}>
```
Comment (ARivas16) [inline src/react/icons/AccessibleBathroom.tsx:6]: This is with `0 0 40 40` is OK @bombozama ??

File: src/react/icons/AccessibleTransportation.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import Icon from '../Icon';
+import { IconComponent } from '../types';
+
+const AccessibleTransportation: IconComponent = (props: any) => (
+    <Icon viewBox="0 0 40 40" {...props}>
```
Comment (ARivas16) [inline src/react/icons/AccessibleTransportation.tsx:6]: The same with this

File: src/react/icons/Culture.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import Icon from '../Icon';
+import { IconComponent } from '../types';
+
+const Culture: IconComponent = (props) => (
+    <Icon viewBox="0 0 40 40" {...props}>
```
Comment (ARivas16) [inline src/react/icons/Culture.tsx:6]: and this

File: src/react/icons/PicnicTable.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Icon from '../Icon';
+import { IconComponent } from '../types';
+
+const Outdoor: IconComponent = (props) => (
+    <Icon viewBox="0 0 40 40" {...props}>
```
Comment (ARivas16) [inline src/react/icons/PicnicTable.tsx:6]: and this

File: src/react/icons/PlacesToStay.tsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Icon from '../Icon';
+import { IconComponent } from '../types';
+
+const PlacesToStay: IconComponent = (props) => (
+    <Icon viewBox="0 0 40 40" {...props}>
```
Comment (ARivas16) [inline src/react/icons/PlacesToStay.tsx:6]: this icon too

File: src/react/icons/ThingsToDo.tsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Icon from '../Icon';
+import { IconComponent } from '../types';
+
+const ThingsToDo: IconComponent = (props) => (
+    <Icon viewBox="0 0 40 40" {...props}>
```
Comment (ARivas16) [inline src/react/icons/ThingsToDo.tsx:6]: This one too

**Pull request merged**